src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (a) => {
+    counter++;
+
+    if (counter > 3 && counter % 2 === 0) {
+      return () => 'Bzzz... Error!';
```

Comment:
```suggestion
      return  'Bzzz... Error!';
```
do you really need to return a function here?

Reply:
![Снимок экрана (38)](https://user-images.githubusercontent.com/72734973/179791683-92f36631-a17a-4a5e-abbb-0d286bfa14de.png)

Reply:
повертай функцію всередині якої буде твій if, ця функція буде приймати друге число та повертати рядок або суму, так буде читабельніше. І варто переіменувати a i b